acquire/utils.py

```diff
@@ -84,6 +84,11 @@ def create_argument_parser(profiles: dict, volatile: dict, modules: dict) -> arg
         action="store_true",
         help="compress output (if supported by the output type)",
     )
+    parser.add_argument(
+        "--compress-method",
+        choices=COMPRESSION_METHODS,
```

Comment:
I think it would be good if `check_and_set_acquire_args()` would check if the given compression method is allowed for the given compression type.